swagger.yaml

```diff
@@ -0,0 +1,51 @@
+openapi: 3.0.1
+info:
+  title: Ainized-Style-Transfer 
+  description:  |
+    # This is an API server that uses deep learning to compose one image in the style of another image
+    ---
+    How to use:
+      * send base image and style image to ainized style-transfer server
```

Comment:
What about just
* pick a base image
* pick a style image
* wait til the result shows up

Reply:
Done!